# Define the goal
geometry_msgs/PoseStamped goal
---
# Define the result
geometry_msgs/PoseStamped final
---
# Define a feedback message
geometry_msgs/PoseStamped current
